webapp/src/language/es.lacchain.json

```diff
@@ -3,9 +3,9 @@
     "/entities>sidebar": "Entidades",
     "/entities>title": "Entities - Panel",
     "/entities>heading": "Entities",
-    "/lacchain/network>sidebar": "LACChain Topologia de rede",
-    "/lacchain/network>title": "LACChain Topologia de rede - Panel",
-    "/lacchain/network>heading": "LACChain Topologia de rede",
+    "/lacchain/network>sidebar": "Topologia de red",
```

Comment:
@tetogomez 

English should read Network topology , please read comment in issue

https://github.com/eoscostarica/eosio-dashboard/issues/403